Cargo.toml

```diff
@@ -72,4 +72,4 @@ fusedev = ["nydus-utils/fusedev", "fuse-backend-rs/fusedev"]
 virtiofs = ["fuse-backend-rs/vhost-user-fs", "vm-memory", "vhost", "vhost-user-backend", "virtio-queue", "virtio-bindings", "blobfs/virtiofs"]
 
 [workspace]
-members = ["api", "app", "error", "rafs", "storage", "utils", "blobfs"]
+members = ["api", "app", "error", "rafs", "storage", "utils", "blobfs", "contrib/nydusd-http-server"]
```

Comment:
Can we make the name more significant? like `nydus-backend-proxy`.

Reply:
I don't think compiling the new binary you added is necessary when compile rafs.
So can we remove it from nydus' cargo toml members?